patches/@react-native-community/cli-platform-android+3.0.0-alpha.2.patch

```diff
@@ -0,0 +1,12 @@
+patch-package
+--- a/node_modules/@react-native-community/cli-platform-android/native_modules.gradle
++++ b/node_modules/@react-native-community/cli-platform-android/native_modules.gradle
+@@ -170,7 +170,7 @@ class ReactNativeModules {
+ 
+     def cmdProcess
+     def root = getReactNativeProjectRoot()
+-    def command = "node ./node_modules/react-native/cli.js config"
++    def command = "npx react-native config"
```

Comment:
What is this about?

Reply:
This was the wrong path to the cli. Tried to be fancy but should work now